### ISSUE TITLE
Process feedback Anne

### ISSUE THREAD

**Issue by richelbilderbeek** [informalr/informalr#62]
From @fastnotfurious:

> 2) I was thinking whether you’d like some support for the meetings? Maybe we could think as a group about sharing workload a bit. One idea might be that everyone who wants to prepares a specific topic for a session that then help us for our collaborative project. 
For example, I find it interesting to scrape data from the web and I could prepare a session devoted to how to do that. 

I should indeed let the group vote for/against this one day.

> 3) Would it be an idea to introduce and use RStudio cloud (https://rstudio.com/resources/webinars/teaching-r-online-with-rstudio-cloud/) for collective learning purposes? (I’m not so sure about the benefits of RStudio cloud, but it seems to make sense). 

 * [ ] Investigate

> 4) I feel that once people (me included! ;D) get a feeling for how to use git, it would be nice to have a collection of resources on how to do simple and basic things in R (covering workflow basics, like why projects make sense https://r4ds.had.co.nz/workflow-projects.html)

She's right. There is a git instruction section [here](https://github.com/richelbilderbeek/git_voor_jonge_tieners), in Dutch.

> 5) Related to that, maybe it makes sense to have some kind of “agenda” to cover topics all of us would like to learn sth about? Right now I know nothing about the experience of most of the people in the team; maybe we could have a session where we find out what people still want to or have to learn before we start with our collaborative project (I’m thinking of basics of data visualisation, tidy data, markdown,…).

I agree with her thoughts here, should vote for such a thing.

**Comment by richelbilderbeek** [comment]
Close for now.
